rules.d/50-udev-default.rules.in

```diff
@@ -96,6 +96,8 @@ KERNEL=="fuse", MODE="0666", OPTIONS+="static_node=fuse"
 # The static_node is required on s390x and ppc (they are using MODULE_ALIAS)
 KERNEL=="kvm", GROUP="kvm", MODE="@DEV_KVM_MODE@", OPTIONS+="static_node=kvm"
 
+KERNEL=="vfio", MODE="0666", OPTIONS+="static_node=vfio/vfio"
```

Comment:
This follows the pattern done with other nodes, e.g. `/dev/vhost-vsock`. But I'm wondering if we're doing the right thing in general. Those nodes are also created by `systemd-tmpfiles-setup-dev.service`, but with different permissions.
```console
$ grep vhost-net /run/tmpfiles.d/static-nodes.conf
c! /dev/vhost-net 0600 - - - 10:238
$ grep vhost-net rules.d/50-udev-default.rules.in
KERNEL=="vhost-net", GROUP="kvm", MODE="@DEV_KVM_MODE@", OPTIONS+="static_node=vhost-net"
$ grep vhost-net build/rules.d/50-udev-default.rules
KERNEL=="vhost-net", GROUP="kvm", MODE="0666", OPTIONS+="static_node=vhost-net"
```

This means that we'll most likely first create the node with more restrictive permissions, and then when the module is actually loaded, we'll adjust. Shouldn't we be using the more restrictive permissions from the beginning? The difference is that the latter allows users to access the node even if the kernel module is not loaded, triggering module insertion in the kernel. @poettering?

(The PR looks OK if we ignore the above, so I guess we can merge it either way.)

Reply:
Hmm, in my testing with just this patch, I was able to `cat /dev/vfio/vfio` as an unpriveleged user, and that triggered module autoloading.

Reply:
And what are the permissions on `/dev/vfio/vfio` before and after the module is loaded?

Reply:
Before the module is loaded: 0666; after: 0666.